src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
According to the code style rules, stylesheets with a preprocessor file extension (like .scss) should not be linked directly in HTML. Instead, compile it to .css and link to the compiled stylesheet.